### ISSUE TITLE
Timeline question Sync

### ISSUE THREAD

**Issue by wingedonezero** [AlexAltea/curator#13]
I came across this project and believe it could save a lot of time. When is the Sync feature planned to be ready to try out? I did see on the main page it wasn't implemented as of yet.

**Comment by AlexAltea** [comment]
I haven't published the code yet as the accuracy is far too bad. I'm working on improving this, but it will take me few more weeks.

**Comment by alanorth** [comment]
Is the plan that this is for synchronizing audio and subtitles? I've been using [ffsubsync](https://github.com/smacke/ffsubsync) for that for years.

**Comment by AlexAltea** [comment]
Yes, for audio-audio and audio-subs sync. My current issues are with video-audio sync.

**Comment by wingedonezero** [comment]
I dont know if this will help. But ive been using a program https://github.com/JorenSix/SyncSink and have had very good luck with it. Its based on another program by the same guy https://github.com/JorenSix/Panako . In which it finds several points and spits out a delay based on multiple points. Now while the way ive been doing it was taking those values and doing all the other steps myself. Ive only had prolly 3 out of a few hundred that didn't sync correctly. Maybe you can take a look and it will help with what your working on.
